src/containers/booklist.js

```diff
@@ -14,18 +15,30 @@ const BookList = ({ books }) => (
     </thead>
     <tbody>
       {books.map(book => (
-        <Book id={book.id} title={book.title} category={book.category} key={book.id} />
+        <Book
+          id={book.id}
+          title={book.title}
+          category={book.category}
+          key={book.id}
+          book={book}
+          removeBook={removeBook}
```

Comment:
* [ ] Then pass `handleRemoveBook` to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)
```suggestion
          removeBook={handleBookRemove}
```